versioned_docs/version-v3.0.x/extend/extend-apiml/api-mediation-infinispan.md

```diff
@@ -68,19 +84,21 @@ Configure Infinispan as a storage solution through the Caching service by settin
         caching-service:
           storage:
             mode: infinispan
-            infinispan: 
+            infinispan:
+              jgroups.port: 7098
               initialHosts: lpar2[7098]
               persistence:
-                dataLocation: /global/zowe/workspace/caching-service/data01
-                indexLocation: /global/zowe/workspace/caching-service/index01
+                dataLocation: /global/zowe/workspace/caching-service/data
+                indexLocation: /global/zowe/workspace/caching-service/index
```

Comment:
add a comment to the example about needing to be in a non-shared filesystem (tendency to copy-paste)